readme.md

```diff
@@ -1,23 +1,39 @@
 # wcag-act-rules
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/f053b1b3-85d4-4105-926b-9b30bebe83c0/deploy-status)](https://app.netlify.com/sites/wai-wcag-act-rules/deploys)
-
+- [![Netlify Status](https://api.netlify.com/api/v1/badges/f053b1b3-85d4-4105-926b-9b30bebe83c0/deploy-status)](https://app.netlify.com/sites/wai-wcag-act-rules/deploys)
 - Preview: https://wai-wcag-act-rules.netlify.app/
-- WAI Website Manual: https://wai-website-theme.netlify.com/overview/
+- WAI Website Manual: https://wai-website-manual.netlify.app/
```

Comment:
@remibetin  Is this still where we want to point people to?

Reply:
@daniel-montalvo I would suggest being more specific in the Readme.

Proposition:

```markdown
- [How to Create and Edit Documentsl](https://wai-website-theme.netlify.app/writing/)
- [Design Components](https://wai-website-theme.netlify.app/components/)
```